### ISSUE TITLE
Move file upload to Amazon S3

### ISSUE THREAD

**Issue by joaofcosta** [cesium/intra.seium.org#86]
Yeah, using an **ephemeral** filesystem has its disadvantages.
Hope this guide helps: https://devcenter.heroku.com/articles/paperclip-s3 .

**Comment by frmendes** [comment]
I would recommend Carrierwave, but since we are already using Paperclip, whoever does this *MUST* upgrade it to 5.x. I spent 1h last night browsing Paperclip's source code until I realized 4.x was bundled, which has some issues with S3 management.

Setting up S3 is rather tricky. The correct way to do it would be creating group and adding a new user to that same group. Then, creating a policy (use [the one mentioned here](https://github.com/thoughtbot/paperclip/wiki/Paperclip-with-Amazon-S3)), going to the group permissions and attaching the new policy.

Include `dotenv-rails` in the project and set a `.env` file with the credentials. *DO NOT COMMIT IT TO GIT*. Instead add it to the `.gitignore` and commit a `.env.sample` file with the variables and a fake string. e.g. `AWS_REGION="your-region"`
